plugins/rtmp-services/data/services.json

```diff
@@ -284,6 +284,27 @@
                 "x264opts": "scenecut=0"
             }
         },
+        {
+            "name": "Luzento.com - RTMP",
+            "common": false,
```

Comment:
False is the default so this line can be omitted.

Reply:
I've removed this line